### PR TITLE
feature: add configurable timesout. defaults to existing values

### PR DESCRIPTION
**Pull request by sumitanvekar**
### :pencil: Description
HTTP and GRPC timeouts were hardcoded. This  feature makes them configurable.
The defaults are set to previously hardcoded values so that this feature does not change behaviour.

### :link: Related Issues

### CODE REVIEW

File: cmd/flags/target.go [hunk 1]
```diff
@@ -24,16 +24,18 @@ import (
 
 // Target stores flags related to the target.
 type Target struct {
-	HTTPHost            string
```
Comment (sumitanvekar) [inline cmd/flags/target.go:27]: Had to add spaces to all variables due to the new name.

File: cmd/flags/target.go [hunk 2]
```diff
@@ -43,8 +45,10 @@ func (t *Target) String() string {
 func (t *Target) initFlags() {
 	flag.StringVar(&t.HTTPHost, "target-http-host", "http://localhost", "HTTP host to warm up")
 	flag.IntVar(&t.HTTPPort, "target-http-port", 8080, "HTTP port for warm up requests")
+	flag.IntVar(&t.HTTPTimeoutMilliseconds, "target-http-timeout-milliseconds", 10000, "HTTP timeout for requests")
```
Comment (sumitanvekar) [inline cmd/flags/target.go:48]: default same as previous hardcoded value. note that the previous one was in seconds.

File: cmd/flags/target.go [hunk 3]
```diff
@@ -43,8 +45,10 @@ func (t *Target) String() string {
...
 	flag.StringVar(&t.GrpcHost, "target-grpc-host", "localhost", "Grpc host to warm up")
 	flag.IntVar(&t.GrpcPort, "target-grpc-port", 50051, "Grpc port for warm up requests")
+	flag.IntVar(&t.GrpcTimeoutMilliseconds, "target-grpc-timeout-milliseconds", 1000, "Grpc timeout for requests")
```
Comment (sumitanvekar) [inline cmd/flags/target.go:51]: same as https://github.com/ExpediaGroup/mittens/pull/297/files#r1225151835

File: docs/about/getting-started.md [hunk 4]
```diff
@@ -11,32 +11,34 @@ The application receives a number of command-line flags including the requests t
...
-| -max-warmup-seconds              | int     | 30                          | Maximum time spent sending warmup requests to the target service. Please note that `max-duration-seconds` may cap this duration                                                                                                                                                         |
-| -concurrency-target-seconds      | int     | 0                           | Time taken to reach expected concurrency. This is useful to ramp up traffic.                                                                                                                                                                                                            |
+| Flag                                                           | Type    | Default value               | Description                                                                                                                                                                                                                                                                             |
```
Comment (sumitanvekar) [inline docs/about/getting-started.md:14]: Column size changed due to the new flag name.

**Review by worldtiki — APPROVED** [review]

**Review by nikos912000 — APPROVED** [review]

**Pull request merged**